packages/cache/src/CacheStorage.ts

```diff
@@ -45,7 +45,7 @@ function getMemoizedHashesFor(cwd: string): { [file: string]: string } {
   );
 
   const results: { [key: string]: string } = {};
-  for (const file in filesInCwd) {
+  for (const file of filesInCwd) {
```

Comment:
Will `for .. of` work if `savedHashOfThisRepo` is an object?

Reply:
Yeah I believe it will still since the `filesInCwd` that is looped here is an array derived from the `savedHashOfThisRepo` using `object.keys`